src/config/i18n.js

```diff
@@ -10,6 +10,7 @@ i18n
   .use(LanguageDetector)
   .use(reactI18nextModule) // if not using I18nextProvider
   .init({
+    lng: 'fr',
```

Comment:
Why do you need to override it?
The fallbackLng below is not enough?

Reply:
If user browser locale is **en**, it is used as _detected language_, but we do not have english translations.

Reply:
> english translations are not fully contributed.

Are they not the default ones?